core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties

```diff
@@ -22,6 +22,5 @@
 
 ExecutorsWarning=Construir no n\u00F3 embutido pode ser um problema de seguran\u00E7a. \
 Voc\u00EA deveria configurar o n\u00FAmero de executores no n\u00F3 embutido para 0. \
-Vide <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">a documenta\u00E7\u00E3o</a>.
```

Comment:
is this right? it seems to be just cutting off the text. there's a \ without any continuation

Reply:
Same thing here... the last sentence was removed although the original has it:

```
ExecutorsWarning=Building on the built-in node can be a security issue. \
  You should set the number of executors on the built-in node to 0. \
  See <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">the documentation</a>.
# TODO(terminology) rename redirect URL
```

Reply:
And after fixed, I executed the translation_tool.pl again:

```
$ git diff
diff --git a/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties b/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
index 8df5b1eea3..7d609c3194 100644
--- a/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
+++ b/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
@@ -22,4 +22,3 @@
 
 ExecutorsWarning=Construir no n\u00F3 embutido pode ser um problema de seguran\u00E7a. \
 Voc\u00EA deveria configurar o n\u00FAmero de executores no n\u00F3 embutido para 0. \
-Vide <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">a documenta\u00E7\u00E3o</a>.
```

It just removed the last line again.